modules/javafx.base/src/test/java/test/javafx/util/converter/LocalDateStringConverterTest.java

```diff
@@ -48,42 +53,100 @@
 public class LocalDateStringConverterTest {
     private static final LocalDate VALID_DATE = LocalDate.of(1985, 1, 12);
```

Comment:
I presume that this is not locale sensitive?

Reply:
LocalDate is not locale sensitive.